__tests__/test-merge.js

```diff
@@ -564,7 +569,135 @@ describe('merge', () => {
     } catch (e) {
       error = e
     }
+    expect(await fs.read(outFile, 'utf-8')).toBe(
+      await fs.read(testFile, 'utf-8')
+    )
     expect(error).not.toBeNull()
     expect(error.code).toBe(Errors.MergeNotSupportedError.code)
   })
+
+  it("merge two branches that modified the same file, custom conflict resolver (prefers ours)'", async () => {
+    // Setup
+    const { fs, gitdir } = await makeFixture('test-merge')
+
+    const commit = (
+      await log({
+        fs,
+        gitdir,
+        depth: 1,
+        ref: 'a-merge-c-recursive-ours',
+      })
+    )[0].commit
+    // Test
+    const report = await merge({
+      fs,
+      gitdir,
+      ours: 'a',
+      theirs: 'c',
+      author: {
+        name: 'Mr. Test',
+        email: 'mrtest@example.com',
+        timestamp: 1262356920,
+        timezoneOffset: -0,
+      },
+      onMergeConflict: ({
+        ourContent,
+        baseContent,
+        theirContent,
+        ourname = 'ours',
+        baseName = 'base',
+        theirName = 'theirs',
+      }) => {
+        const LINEBREAKS = /^.*(\r?\n|$)/gm
+        const ours = ourContent.match(LINEBREAKS)
```

Comment:
What this line is doing? It's hard to understand. Is it the same as `ourContent.split(/\r?\n/)`?

Reply:
This regex is copied from the original `mergeFile` implementation, but yes it's essentially equivalent to `ourContent.split(/\r?\n/)`